.zenodo.json

```diff
@@ -167,6 +170,9 @@
       "name": "Raamana, Pradeep Reddy",
       "orcid": "0000-0003-4662-0558"
     },
+    {
+      "name": "Klug, Julian"
```

Comment:
@JulianKlug I've added you to the Zenodo fragment, and you will be listed as an author in the [Zenodo archive](https://doi.org/10.5281/zenodo.591597). Please let me know if you would like your name to be rendered otherwise, or if you have an affiliation and/or [ORCID](https://orcid.org/) you would to use. I can also remove you if you'd rather not be listed as an author.

Reply:
sorry for the late reply, below my affiliation details
ORCID: 0000-0002-4849-9811
affiliation: University of Geneva, Switzerland